dokka-integration-tests/utilities/src/main/kotlin/org/jetbrains/dokka/it/processUtils.kt

```diff
@@ -1,55 +1,23 @@
 /*
  * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
-
 package org.jetbrains.dokka.it
 
-import kotlinx.coroutines.CompletableDeferred
-import kotlinx.coroutines.async
-import kotlinx.coroutines.runBlocking
-import kotlin.concurrent.thread
+import java.util.concurrent.TimeUnit
 
 class ProcessResult(
     val exitCode: Int,
     val output: String
 )
 
-fun Process.awaitProcessResult(): ProcessResult = runBlocking {
-    val exitCode = async { awaitExitCode() }
-    val output = async { awaitOutput() }
-    ProcessResult(
-        exitCode.await(),
-        output.await()
-    )
-}
+fun Process.awaitProcessResult(): ProcessResult {
+    waitFor(60, TimeUnit.SECONDS)
 
-private suspend fun Process.awaitExitCode(): Int {
-    val deferred = CompletableDeferred<Int>()
-    thread {
-        try {
-            deferred.complete(this.waitFor())
-        } catch (e: Throwable) {
-            deferred.completeExceptionally(e)
-        }
-    }
+    val output = inputStream.reader().readText()
+    println(output)
```

Comment:
minor: AFAIU, previously output was printed to `stdout` incrementally, but now it will be printed fully after the full execution, is it correct? this could be not ideal when running locally if the test takes long enough, and there is no any output